test/verify/check-storage-ignored

```diff
@@ -33,17 +33,17 @@ class TestStorageIgnored(StorageCase):
         disk = self.add_ram_disk()
         b.wait_in_text("#drives", disk)
 
-        m.execute(f"yes | mke2fs -q -L TEST {disk}")
+        m.execute(f"yes | mke2fs -q -L TESTLABEL {disk}")
         with b.wait_timeout(30):
-            b.wait_in_text("#mounts", disk)
+            b.wait_in_text("#mounts", "TESTLABEL")
```

Comment:
should this be now `disk + "(TESTLABEL)"`?

Reply:
Yes, why not!